app/src/main/AndroidManifest.xml

```diff
@@ -18,7 +18,11 @@
     <activity
       android:name="com.alexstyl.themingandroid.advanced.AdvancedExampleActivity"
       android:label="@string/Advanced_Example"
-      android:parentActivityName="com.alexstyl.themingandroid.simple.SimpleExampleActivity" />
+      android:parentActivityName="com.alexstyl.themingandroid.simple.SimpleExampleActivity">
+    <meta-data
+        android:name="android.support.PARENT_ACTIVITY"
+        android:value="com.alexstyl.themingandroid.simple.SimpleExampleActivity"/>
+    </activity>
```

Comment:
There are plans of introducing more samples in the future. Supporting such an old version of the OS for a sample project is not my thing :P